web/ui/templates/status.html

```diff
@@ -45,6 +45,14 @@ <h2 id="runtime">Runtime Information</h2>
           <th>GOGC</th>
           <td>{{.GOGC}}</td>
         </tr>
+        <tr>
+          <th>Storage Time Retention</th>
+          <td>{{.RetentionTime}}</td>
+        </tr>
+        <tr>
+          <th>Storage Size Retention</th>
+          <td>{{.RetentionSize}}</td>
+        </tr>
```

Comment:
Can we simplify this and have just one line?

(exepmle values that would be. printed: 
- 15d
- 15d or 10GB
- 19GB
)

Reply:
yep , not a bad idea. I updated the code.

Reply:
![Screenshot_20190311_110834](https://user-images.githubusercontent.com/8903888/54112108-1ed40680-43ee-11e9-94ca-e66b23a5cd95.png)
![Screenshot_20190311_110908](https://user-images.githubusercontent.com/8903888/54112109-1f6c9d00-43ee-11e9-8d05-e5bcd92ea216.png)